react-components/README.md

```diff
@@ -0,0 +1,44 @@
+# CIF Core Components - React Components
+
+The React Components project is the code base for all the CIF Core Components built using [React](https://reactjs.org/). The following components are part of this project:
+
+-   Minicart
+-   Checkout
+-   Sign In
+-   My Account
+-   Password recovery (Forgot Password)
+
+## Building the project
+
+The project is built using the command `npm run build`. The build process bundles all the code into one client library which is placed in `../ui.apps/src/main/content/jcr_root/apps/core/cif/clientlibs/react-components/dist`.
```

Comment:
I have two main concerns with that entire change, but maybe I missed something:
- it seems this project is not integrated with the main "reactor" build, but it must also be built, otherwise the clientlib is going to be missing in ui.apps, right?
- I then find it a bit "tricky" to build something and add it to another project (ui.apps), because it basically creates a dependency that isn't reflected now at the project level ... can we somehow fix that? At least to make sure that the reactor project builds everything in the right order?

Reply:
The `react-components` project is built by the `ui.apps` project, see https://github.com/adobe/aem-core-cif-components/pull/139/files#diff-d6ca0a35a41bb1932eddf863e9f17311R199

Reply:
ah ok, sorry, I thought you had also removed that and really wanted the two sub-projects to be independent.